webapp/src/components/pulsating_dot/index.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react'
+
+import './pulsating_dot.scss'
+import {Coords} from '../tutorial_tour_tip/tutorial_tour_tip_backdrop'
+
+type Props = {
+    targetRef?: React.RefObject<HTMLImageElement>;
+    className?: string;
+    onClick?: (e: React.MouseEvent) => void;
+    coords?: Coords;
+}
+
+export class PulsatingDot extends React.PureComponent<Props> {
```

Comment:
Why is this a class component?

Reply:
It's copy-pasted from Mattermst webapp. I don't want to make any changes to it and make it a Board's component. Copy this component was required because we don't have any good way of sharing components right now. I want to leave it as it is.

Thoughts?

Reply:
Actually, I prefer to do it all the way around, we want channels to migrate more into functional components whenever makes sense, and in this cases, looks like makes a lot of sense. So I would suggest to change this to functional component and make a PR in channels to copy this over there.

Reply:
Fixed in Boards for now.